### PR TITLE
Publish solution for lab 5

### PR DESCRIPTION
**Pull request by teodutu**

### CODE REVIEW

**Comment by sketch873** [comment]
In the lab05/skel/lin/ some Makefiles do not have PHONY all

Let only one newline between test.o and clean: lab05/skel/lin/9-my_malloc/Makefile
Leading space: lab05/skel/lin/9-my_malloc/Makefile (libmy_malloc.so)
Let only one newline between macros and main: lab05/sol/lin/3-alloc/alloc.c
Let only one newline between block comments: lab05/skel/win/malloc-wrapper/test.c AND lab05 /sol/win/malloc-wrapper/test.c

There are two years in the descriptions: 2006 and 2011. My suggestion is to remove the year from all files.

File: labs/lab05/sol/lin/4-gdb/fault.c [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	}
+}
+
+int main(void)
+{
+	char *message;
+
+	message = read_message();
+
+	upper(message);
```
Comment (sketch873) [inline labs/lab05/sol/lin/4-gdb/fault.c]: upper_string(message);

File: labs/lab05/sol/lin/4-gdb/fault.c [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+	upper_string("Upper string is: %s\n", message);
```
Comment (sketch873) [inline labs/lab05/sol/lin/4-gdb/fault.c]: printf("Upper string is: %s\n", message);

File: labs/lab05/sol/lin/7-trim/trim.c [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ * Lab #5
+ *
+ * Task #7, lin
+ *
+ * mcheck usage
+ */
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
```
Comment (sketch873) [inline labs/lab05/sol/lin/7-trim/trim.c:12]: #include "utils.h"

**Review by sketch873 — CHANGES_REQUESTED** [review]

**Review by sketch873 — APPROVED** [review]

**Review by lucigrigo — APPROVED** [review]

**Pull request merged**